performance/lib/jsonify/jsonify.js

```diff
@@ -0,0 +1,258 @@
+// copied from:
+//  https://github.com/substack/jsonify/blob/master/lib/parse.js
```

Comment:
Use links from the specific version to emphasize version + license (public domain)

https://github.com/substack/jsonify/blob/7064ab53e5f73fa31e2ce7aa47d210c539662f16/readme.markdown#license

https://github.com/substack/jsonify/blob/7064ab53e5f73fa31e2ce7aa47d210c539662f16/lib/parse.js